draft-ietf-tcpm-converters.mkd

```diff
@@ -1,5 +1,10 @@
 ---
 title: 0-RTT TCP Convert Protocol
+<!---
+On first read, I got confused between the "Converter Protocol" and "Convert".
+I missed the line saying Convert is the short version. And the title says
+"Convert Protocol". I would just use the full name in the title: "The Converter Protocol". 
+-->
```

Comment:
The title of the document says: "Convert Protocol". 

I suggest we use that wording in the document.